shenyu-plugin/shenyu-plugin-brpc/src/main/java/org/apache/shenyu/plugin/brpc/proxy/BrpcProxyService.java

```diff
@@ -82,9 +67,8 @@ public Mono<Object> genericInvoker(final String body, final MetaData metaData, f
                 params[i] = bodyMap.get(brpcParamInfo.getParamNames()[i]).toString();
             }
         }
-        initThreadPool();
-        //todo use com.baidu.cloud.starlight.api.rpc.threadpool.ThreadPoolFactory impl it
-        CompletableFuture<Object> future = CompletableFuture.supplyAsync(() -> getValue(metaData, params), threadPool);
+        ThreadPoolFactory threadFactory = ExtensionLoader.getInstance(ThreadPoolFactory.class).getExtension("shard");
```

Comment:
rename `shard` to `shared` and add a `org.apache.shenyu.plugin.brpc.threadpool.SharedThreadPoolFactory` impl, also need for SPI file.

check `com.baidu.cloud.starlight.core.rpc.threadpool.RpcThreadPoolFactory` for more details.

Reply:
thanks,i have fix it,please help me review it again. @dragon-zhang